lib/src/card/Card.stories.tsx

```diff
@@ -0,0 +1,182 @@
+        Xlarge
+      </DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcCard imageSrc={imagePath} imagePadding="xxlarge" imageCover>
+        Xxlarge
+      </DxcCard>
+    </ExampleContainer>
+    <Title title="Content padding" theme="light" level={2} />
+    <ExampleContainer>
+      <Title title="Xxsmall" theme="light" level={4} />
+      <DxcCard contentPadding="xxsmall">Xxsmall</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xsmall" theme="light" level={4} />
+      <DxcCard contentPadding="xsmall">Xsmall</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Small" theme="light" level={4} />
+      <DxcCard contentPadding="small">Small</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Medium" theme="light" level={4} />
+      <DxcCard contentPadding="medium">Medium</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcCard contentPadding="large">Large</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcCard contentPadding="xlarge">Xlarge</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcCard contentPadding="xxlarge">Xxlarge</DxcCard>
+    </ExampleContainer>
+    <Title title="Margin" theme="light" level={2} />
+    <ExampleContainer>
+      <Title title="Xxsmall" theme="light" level={4} />
+      <DxcCard margin="xxsmall">Xxsmall</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xsmall" theme="light" level={4} />
+      <DxcCard margin="xsmall">Xsmall</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Small" theme="light" level={4} />
+      <DxcCard margin="small">Small</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Medium" theme="light" level={4} />
+      <DxcCard margin="medium">Medium</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcCard margin="large">Large</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcCard margin="xlarge">Xlarge</DxcCard>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcCard margin="xxlarge">Xxlarge</DxcCard>
+    </ExampleContainer>
+  </>
+);
+
+export const Chromatic = Card.bind({});
+Chromatic.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  await userEvent.tab();
+  await userEvent.tab();
+  await userEvent.hover(canvas.getAllByRole("link")[2]);
+  await userEvent.hover(canvas.getAllByText("Hovered default with action")[1]);
+};
```

Comment:
The hover tests should be done in different stories, because we are using play functions (so you don't need the tab function).
The rest of the tests should not be inside the play function. Please check the stories of the other components.

Reply:
The tab function is used for the focused card. I don't understand why do we need to separate the play function into another story.

Reply:
The play function is used for the tests that need some interactions, as you can read in the documentation. https://www.chromatic.com/docs/hoverfocus. Here you are adding all the tests inside a play function, that is not needed. We are following this approach (as the documentation says) for all the components. Also, the focus should be done using the focus function, as you can check also there, and not using the tab.